### DISCUSSION TITLE
Welcome to epd_display Discussions!

### DISCUSSION THREAD

**Discussion by txoof** [txoof/epd_display#56] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by txoof** [comment]
@aaronr8684 
I'm working on restructuring PaperPi so I can remove PyInstaller from the chain. This should make it fairly simple for users to install on any Debian based platform that has Python3. The solution I'm working on builds a pipenv and installs modules on demand depending on the plugins present. This should also make it easier to add external plugins without the need to directly freeze them into the project.

Would you be interested in helping test/develop this? I've got an 80% solution, but could use another brain to help sort out some of the details and make sure it "works" properly for end users and developers that want to build plugins.

**Reply by aaronr8684** [reply]
Yea that sounds much cleaner. Let me know how I can help. I can spin up VM boxes pretty quickly to make sure we have a fresh test environment and then run it on one of my Pi's to confirm functionality.

**Reply by aaronr8684** [reply]
Also I think it was you that had some playbooks to set up a Dev environment. I've been meaning to get an ansible setup at home so I could test those as well if you've updated them to match the new process

**Reply by txoof** [reply]
@aaronr8684 I'd love another brain on this. I'll add you to the private repo where I'm working on this.

Here are the ansible playbooks and some rough documentation on using them: https://github.com/txoof/ansible

**Reply by txoof** [reply]
Here's a link to the private repo: https://github.com/txoof/PaperPi

I'll start a discussion over there with the things I'm working on and start a to_do.md tomorrow. It's late here and it's been a long week.
